PhysicsTools/Heppy/python/analyzers/core/AutoHandle.py

```diff
@@ -7,12 +7,13 @@ class AutoHandle( Handle, object ):
 
     handles = {}
     
-    def __init__(self, label, type, mayFail=False, fallbackLabel=None):
+    def __init__(self, label, type, mayFail=False, fallbackLabel=None, lazy=True):
         '''Note: label can be a tuple : (module_label, collection_label, process)'''
         self.label = label
         self.fallbackLabel = fallbackLabel
         self.type = type
         self.mayFail = mayFail
+	self.lazy = lazy
```

Comment:
Ok, the file was a bit messy because some lines where indented with spaces and other with tabs.
Now I made it all spaces, so that it should show the same in any viewer